docs/data/material/components/drawers/drawers.md

```diff
@@ -134,3 +134,11 @@ Apps focused on information consumption that use a left-to-right hierarchy.
 Apps focused on productivity that require balance across the screen.
 
 {{"demo": "ClippedDrawer.js", "iframe": true}}
+
+## Experimental APIs
```

Comment:
Framing this as an experiment API feels odd. I can understand why for the others, these are things that should arguably be built-in features of the core components. But this one feels too high-level and opinionated to fit into Material UI product/brand.

Reply:
We intentionally avoided adding code snippets that weren't "`@mui/material` native" to not confuse users. A demo without a snippet isn't as useful as the feature is in the API shape. Not sure this fully makes sense though.